nuxeo-arender-rendition-apb/templates/arender_document_converter_dc.yml.j2

```diff
@@ -43,13 +43,16 @@ spec:
           - name: logs
             mountPath: /logs
       containers:
-        - command:
+        - name: arender-document-converter
+{% if arender_rendition_image_version | regex_replace('\.NX.*','') is version('4.4.2', '<') %}
```

Comment:
Instead of having this condition all over the code, we could introduce a variable `before_4_4_2` in `vars/main.yaml`